packages/video_player/video_player_android/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.8.14
```

Comment:
You should revert the CHANGELOG and version change in `video_player_android`; the changes here are only to an unpublished file.